migration_scripts/bulk_load/src/db.clj

```diff
@@ -67,6 +69,31 @@
         filter]
        (jdbc/execute! db)))
 
+(defn explain-analyze!
+  "EXPLAIN ANALYZE a query.
+
+  Returns the execution and planning time of the query on success, or
+  the keyword :timeout if it was not possible to finish the operation
+  within the alotted time."
+  [db timeout [query & binds]]
+  (let [QUERY-PLAN (keyword "QUERY PLAN")
+        key (fn [key] (-> key s/lower-case (s/replace #"\s+" "-") keyword))
+        extract
+        (fn [{e :execution-time p :planning-time {r :actual-rows} :plan}]
+          {:execution-time e :planning-time p :actual-rows r})]
+    (try
+      (as-> query %
+        (format "EXPLAIN (ANALYZE, FORMAT JSON) %s" %)
```

Comment:
we can also add a buffers `EXPLAIN (ANALYZE, BUFFERS, FORMAT JSON)` to understand cache hits and misses, I/O time, and so on

Reply:
In #11 I split this function into two -- one that uses the JSON format from which I pick out only a couple of rows, and then a human-readable version. I've made this change to the human readable version, in that  PR!